tests/js/phet-converter-tests.js

```diff
@@ -0,0 +1,227 @@
+            args: ["{that}", "{arguments}.0.0", "{arguments}.1"]
+        }
+    }
+});
+
+
+fluid.defaults("phetosc.tests.withBundledConverter", {
+    gradeNames: "fluid.component",
+
+    components: {
+        converter: {
+            options: {
+                bundleParameters: true
+            }
+        }
+    }
+});
+
+fluid.defaults("phetosc.tests.withUnbundledConverter", {
+    gradeNames: "fluid.component",
+
+    components: {
+        converter: {
+            options: {
+                bundleParameters: false
+            }
+        }
+    }
+});
+
+fluid.defaults("phetosc.tests.converter.primitiveValueChanged", {
+    gradeNames: [
+        "phetosc.tests.withBundledConverter",
+        "phetosc.tests.deepEqConversionTester"
+    ],
+
+    moduleName: "Primitive parameters, bundled",
+
+    phetPath: "../data/phet/phet-particle-count-changed.json",
+    oscPath: "../data/osc/osc-particle-count-changed-bundled.json",
+
+    components: {
+        converter: {
+            options: {
+                bundleParameters: true
+            }
+        }
+    }
+});
+
+phetosc.tests.converter.runTests = function (that, phetEvent, expected) {
+    jqUnit.test("Conversion from PhET event to OSC messages", function () {
+        var actual = that.converter.toOSC(phetEvent);
+        jqUnit.assertDeepEq("PhET event was correctly converted to OSC messages",
+            expected, actual);
+    });
+};
+
+fluid.defaults("phetosc.tests.converter.primitiveUnbundledValueChange", {
+    gradeNames: [
+        "phetosc.tests.converter.primitiveValueChanged",
+        "phetosc.tests.withUnbundledConverter"
+    ],
+
+    moduleName: "Primitive parameters, not bundled",
+    oscPath: "../data/osc/osc-particle-count-changed-unbundled.json"
+});
+
+fluid.defaults("phetosc.tests.converter.objectParameter", {
+    gradeNames: [
+        "phetosc.tests.withBundledConverter",
+        "phetosc.tests.deepEqConversionTester"
+    ],
+
+    moduleName: "Object-valued parameters, bundled",
+
+    phetPath: "../data/phet/phet-object-valued-parameter.json",
+    oscPath: "../data/osc/osc-object-valued-parameter-bundled.json"
+});
+
+fluid.defaults("phetosc.tests.converter.compositeEvent", {
+    gradeNames: [
+        "phetosc.tests.withBundledConverter",
+        "phetosc.tests.deepEqConversionTester"
+    ],
+
+    moduleName: "Composite event with children, bundled",
+
+    phetPath: "../data/phet/phet-composite-event-drag-ended.json",
+    oscPath: "../data/osc/osc-composite-event-drag-ended-bundled.json"
+});
+
+fluid.defaults("phetosc.tests.withIDPatterns", {
+    gradeNames: "fluid.component",
+
+    components: {
+        converter: {
+            options: {
+                components: {
+                    filter: {
+                        type: "phetosc.eventFilter",
+                        options: {
+                            phetioIDPatterns: "{withIDPatterns}.options.phetioIDPatterns"
```

Comment:
Just an options distribution for this option required? I think the component skeleton is already present?